sources.nix

```diff
@@ -1,8 +1,8 @@
 {
   unstable = builtins.fetchTarball {
     name = "nixos-unstable-2021-10-16";
```

Comment:
Please update the date since it is easier to see how recent the sources are at a glance that way